plugin/audit_log/tests/mtr/audit_log_rotate_with_flush.test

```diff
@@ -0,0 +1,26 @@
+--echo #
+--echo # PS-1098: 'SET GLOBAL audit_log_flush=1' reopens wrong file after auto rotation
+--echo #
+
+--disable_result_log
+--disable_query_log
+--source audit_log_events.inc
+--enable_query_log
+--enable_result_log
+
+SET GLOBAL audit_log_flush=1;
+
+let $MYSQLD_DATADIR= `select @@datadir`;
+let MYSQLD_DATADIR= $MYSQLD_DATADIR;
```

Comment:
Declaring just
```
let MYSQLD_DATADIR = `select @@datadir`;
```
is enough here. Environment variables can still be accessed from MTR via `$MYSQLD_DATADIR`

Reply:
Done